src/scripts/modules/media/tabs/contents/toc/modals/section-name/section-name.coffee

```diff
@@ -0,0 +1,29 @@
+define (require) ->
+  $ = require('jquery')
+  _ = require('underscore')
+  BaseView = require('cs!helpers/backbone/views/base')
+  require('bootstrapModal')
+  require('less!./section-name')
+  template = require('hbs!./section-name-template')
+
+  return class SectionNameModal extends BaseView
+    template: template
+
+    events:
+      'click .btn-ok': 'ok'
+      'keydown': 'keyAction'
+
+    keyAction: (event) ->
+      if event.keyCode == 13
```

Comment:
`if event.keyCode is 13` (which in JavaScript will translate to `===`... never ever use `==` in JS/CS unless you really really really mean it, and even then, maybe don't).